gui/src/installer/view.rs

```diff
@@ -787,39 +788,93 @@ pub fn help_backup<'a>() -> Element<'a, Message> {
 pub fn define_bitcoin<'a>(
     progress: (usize, usize),
     address: &form::Value<String>,
-    cookie_path: &form::Value<String>,
+    rpc_auth_vals: &RpcAuthValues,
+    selected_auth_type: &RpcAuthType,
     is_running: Option<&Result<(), Error>>,
 ) -> Element<'a, Message> {
     let col_address = Column::new()
         .push(text("Address:").bold())
         .push(
             form::Form::new_trimmed("Address", address, |msg| {
-                Message::DefineBitcoind(message::DefineBitcoind::AddressEdited(msg))
+                Message::DefineBitcoind(message::DefineBitcoind::ConfigFieldEdited(
+                    ConfigField::Address,
+                    msg,
+                ))
             })
             .warning("Please enter correct address")
             .size(20)
             .padding(10),
         )
         .spacing(10);
 
-    let col_cookie = Column::new()
-        .push(text("Cookie path:").bold())
+    let col_auth = Column::new()
         .push(
-            form::Form::new_trimmed("Cookie path", cookie_path, |msg| {
-                Message::DefineBitcoind(message::DefineBitcoind::CookiePathEdited(msg))
-            })
-            .warning("Please enter correct path")
-            .size(20)
-            .padding(10),
+            [RpcAuthType::CookieFile, RpcAuthType::UserPass]
+                .iter()
+                .fold(
+                    Row::new()
+                        .push(text("RPC authentication:").small().bold())
+                        .spacing(10),
+                    |row, auth_type| {
+                        row.push(radio(
+                            format!("{}", auth_type),
+                            *auth_type,
+                            Some(*selected_auth_type),
+                            |new_selection| {
+                                Message::DefineBitcoind(
+                                    message::DefineBitcoind::RpcAuthTypeSelected(new_selection),
+                                )
+                            },
+                        ))
+                        .spacing(30)
+                        .align_items(Alignment::Center)
+                    },
+                ),
```

Comment:
Same comment regarding the radio buttons here:
![image](https://github.com/wizardsardine/liana/assets/22457751/13cc7ae8-18ff-42b3-8f20-d2f121cdb6e3)


They should have the same style as the other buttons around, i think.